frontend/src/scenes/data-pipelines/batch-exports/BatchExportEditForm.tsx

```diff
@@ -156,11 +157,16 @@ export function BatchExportsEditFields({
                         </LemonField>
 
                         <div className="flex gap-4">
-                            <LemonField name="file_format" label="Format" className="flex-1">
+                            <LemonField
+                                name="file_format"
+                                label="Format"
+                                className="flex-1"
+                                info="Parquet with zstd compression has shown to offer the best performance"
```

Comment:
nit: Maybe a bit more "recommendation style" like: "We recommend Parquet with zstd compression for the best performance".

very subjective though.